src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java

```diff
@@ -90,4 +93,57 @@ void should_warn_about_missing_classification() {
         assertThat(output.isLeft()).isTrue();
         assertThat(output.getLeft()).containsOnly("Missing journeyClassification");
     }
+
+    @Test
+    void should_report_errors_when_data_is_in_invalid_form() {
```

Comment:
should we have two case one when ocr is invalid and other when we can't let's say parse scanned docs ?

Reply:
Could be just thought it's fairly bulky to set up requirements and test things. Later on 2 more components are coming in to play. Quite possible entire validation code can be extracted to component the way it is now so it doesn't end up like _attach to case_ callback service

tl;dr - will split for clarity